platforms/software/build-init-specs-api/src/main/java/org/gradle/buildinit/specs/BuildInitSpec.java

```diff
@@ -42,9 +41,10 @@ public interface BuildInitSpec extends Describable {
      * @since 8.12
      */
     @Override
+    @SuppressWarnings("deprecation")
     default String getDisplayName() {
         String spaced = getType().replace("-", " ");
-        return WordUtils.capitalizeFully(spaced);
+        return org.apache.commons.lang3.text.WordUtils.capitalizeFully(spaced);
```

Comment:
Can we switch to `commons-text` instead of suppressing the deprecation?

Reply:
I separated the deprecation suppressions into a separate commit and added an explanation.

We don't currently depend on `commons-text` and doing so would add ~250KiB to our distribution again.
We only use 3 deprecated things and I suppose they won't be removed until v4 - and I'm not aware of any plans for a v4 of `commons-lang`.

So unless there is a strong desire to switch I'd avoid the added weight of `commons-text`.